app/assets/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "assets",
+  "dependencies": {
+    "@popperjs/core": "^2.11.8",
```

Comment:
popper makes it so the hovercard (where the check details are rendered on hover) collides with the edges of the screen